app/mockServices/mockConstants.service.ts

```diff
@@ -4,6 +4,14 @@ export class ConstantsService {
   public constants: any;
   constructor () {
     let win: any = window;
+
+    let publisherSynonyms = {
+      'Vendor B': 'Vendor B, Inc.',
+      'Vendor B, Inc': 'Vendor B, Inc.',
+      'Vendor B, Co.': 'Vendor B, Inc.'
+    };
+    _.set(win, 'OPENSHIFT_CONSTANTS.PUBLISHER_SYNONYMS', publisherSynonyms);
```

Comment:
Do you mind adding an empty value to constants.ts with a comment explaining what it is. We might even add the Red Hat values from #429 

It's nice to keep the constants files well commented since it's open source so that people can see all the options together.

Reply:
Done